mutatex/core.py

```diff
@@ -1019,20 +1034,26 @@ class FoldXRepairRun(FoldXRun):
     def check_status(self):
         """
         checks status of the repair run. Since we are just interested in the
-        repaired PDB, two statuses are possible: either the output PDB is
-        present and the run was thus sucessful or not.
+        repaired PDB, tree statuses are possible: 1) the output PDB is
+        present and the run was thus sucessful, 2) the output PDB was generated
+        with unrezognized molecules, or 3) the run was not sucessful.
         Returns
         ----------
         str
-           either "already_done", if the output PDB is present, or "not_done"
-           otherwise
+           either "already_done", if the output PDB is present, "broken" if 
+           unrecognized molecules found or "not_done" otherwise
         """
         if os.path.exists(self.working_directory):
             log.warning("working directory %s already exists." % self.working_directory)
             if self.foldx_version.repair_pdb_output_fname(self.pdbs[0]) in os.listdir(self.working_directory):
-                log.warning("PDB output file already present; run %s will be skipped" % self.name)
-                return "already_done"
-        return "not_done"
+               if 'Unrecognized_molecules.txt' in os.listdir(self.working_directory): 
+                   log.warning("A previous PDB repair was run with unrecognized molecules for %s." % self.name)
```

Comment:
can you also add "it will be rerun" similar to what we have in other cases (e.g. mutation runs)?